config/environment.rb

```diff
@@ -4,3 +4,15 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+# Set up ActionMailer for SMTP
+
+ActionMailer::Base.smtp_settings = {
+  :address        => ENV["SMTP_SERVER"],
+  :domain         => ENV["EMAIL_DOMAIN"],
+  :port           => 587,
```

Comment:
@byronfactual - I made this change, but shouldn't it be `ENV["SMTP_PORT"].blank? ? DEFAULT_SMTP_PORT : ENV["SMTP_PORT"].to_i `?